### PR TITLE
Update project to .net standard 2.1 and also remove all net framework…

### PR DESCRIPTION
**Pull request by Reapism**
### Upgrade solution and test project properly

This seemed to be an open issue for a while, and I want to contribute. I think its necessary to have a framework like this target netstandard. I have been able to correct this with the following commit.

- removed packages config as its no longer necessary.
- removed old project file.
- Updated UltraMapper.Tests to .NETCore as it is required for the VS test explorer.
- Upgrade to .netstandard 2.1

The issue was the underlying NET Framework dependencies. This removes all .NET framework dependencies, and replaces the project with .net standard 2.1. The test project multi-targets .net core 3.1 and .net standard. 

The multi-target is necessary to be able to run the tests via visual studio. 

Please consider reviewing/merging this PR. It's a big change for the users of this project.

### Proof of working concept
1. Tests - they all pass
image
2. Referencing the local nuget package in a netcore project resolving correctly. 
image

Linked Issues
#1 
#5

### CODE REVIEW

File: UltraMapper.Tests/UltraMapper.Tests.csproj [hunk 1]
```diff
@@ -1,34 +1,20 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472</TargetFrameworks>
```
Comment (Reapism) [inline UltraMapper.Tests/UltraMapper.Tests.csproj:4]: I don't think Test projects should generate NuGet packages, so the section was removed. They don't get deployed anyways. Let me know if this is a valid concern

Comment (maurosampietro) [reply UltraMapper.Tests/UltraMapper.Tests.csproj:4]: That property is needed to inform what framework to compile against.

**Comment by maurosampietro** [comment]
Hi, can you confirm you can now build the project against both net472 and standard2.1?

**Comment by maurosampietro** [comment]
I faced problems trying to build and release the relative packages with appveyor months ago but i'm trying again now

File: UltraMapper.Tests/UltraMapper.Tests.csproj (hunk 1 above)
Comment (Reapism) [reply UltraMapper.Tests/UltraMapper.Tests.csproj:4]: I meant the nuget package section. That property has been removed as it's a multi target

**Comment by Reapism** [comment]
> Hi, can you confirm you can now build the project against both net472 and standard2.1?

I will confirm for net472 as soon as I can. .net standard it absolutely does. It wouldn't be able to compile .net core without that

**Comment by Reapism** [comment]
> I faced problems trying to build and release the relative packages with appveyor months ago but i'm trying again now

You may need to reconfigure it to deploy properly. I removed all references to .net framework as its unneeded and unnecessarily makes it difficult to use such a tool

**Comment by maurosampietro** [comment]
A netstandard nuget package is finally available both here on github and on nuget.

**Comment by Reapism** [comment]
Was this merged?

**Comment by maurosampietro** [comment]
It was not merged, but the goal is achieved

**Pull request closed without merging**